### DISCUSSION TITLE
Accessing Android/media

### DISCUSSION THREAD

**Discussion by Gaijw78282** [d4rken-org/sdmaid-se#1432] [Q&A] [answered]
Hello I come here to suggest to have put an option to show hidden files of the system because often these files save data that increases over time like those of WhatsApp that is in the android folder > media that is hidden and does not show in the SD app anymore.

**Comment by d4rken** [comment]
Can you elaborate? If the permissions allow, SD Maid already shows files in Android/media.

You can browse there with the Storage Analyzer:


Analyzer > Apps > WhatsApp > App Media 

![Screenshot from 2024-10-10 10-37-29](https://github.com/user-attachments/assets/22b82d5a-4898-4236-a75c-777aa79a1583)

**Comment by d4rken** [comment]
From https://github.com/d4rken-org/sdmaid-se/issues/1431#issue-2578649691

https://github.com/user-attachments/assets/a92e22d3-7a5a-46ec-91a5-e646c461c1c6

https://github.com/user-attachments/assets/d9533175-c9ea-4fff-9b31-6f6409031ff6

https://github.com/user-attachments/assets/a92e22d3-7a5a-46ec-91a5-e646c461c1c6


Uploading Screen_Recording_20241010_090127_My Files.mp4…

 In these videos shows in the app where I can not find the hidden folder and in the other shows my app file administrator my phone and S23 5G android 14 this watsapp folder is in the android directory and in the SD analyzer but does not show this folder.

**Comment by d4rken** [comment, marked as answer]
You are trying to access the folder the wrong way.

You are going:

Analyzer > Storage > User Files > ...

but all files under `Android/media` belong to specific apps. So you have to go

Analyzer > Storage > Apps > WhatsApp > App Media > ...

**Comment by Gaijw78282** [comment]
Sim obrigado eu achei aqui.

![Screenshot_20241010_102942_ES File Explorer](https://github.com/user-attachments/assets/d7cb3475-637f-45e4-a3a3-6b0ecf3cb8f0)
![Screenshot_20241010_102950_ES File Explorer](https://github.com/user-attachments/assets/d7af7f22-2f94-44df-bbf1-dd96f84c1d39)
![Screenshot_20241010_102957_ES File Explorer](https://github.com/user-attachments/assets/3fa2d6e6-d946-408b-9e0a-48048eb72edf)
![Screenshot_20241010_103002_ES File Explorer](https://github.com/user-attachments/assets/7c862b1a-cf7f-4f55-abd6-6ad4ad4684f1)
E que estou acostumado com Es file Explorer onde ja mostra no começo o quantitativo na pasta oculta dando meio que uma simplicidade. Mais obrigado pela paciência e desculpe qualquer coisa.

**Comment by Gaijw78282** [comment]
E por que seria bom mostrando na quela parte do analizador arquivos ocultos conseguisse uma forma de mostrar por lá sem precisar is buscar no app seria maravilhoso.
![Screenshot_20241010_110814_SD Maid SE](https://github.com/user-attachments/assets/ea8394f8-40cd-44cc-bba6-1d322f008c60)
 E tivesse uma função que desse pra automatizar a exclusão desses arquivos em horários programados. Tipo vc entrava na pasta clicaa e segurava e colocava pra agendar exclusão.
